src/IECoreMaya/FromMayaSkinClusterWeightsConverter.cpp

```diff
@@ -66,8 +66,14 @@ FromMayaSkinClusterWeightsConverter::FromMayaSkinClusterWeightsConverter( const
 		"Compress weights as Shorts",
 		true
 	);
+	m_compressionThreshold = new IECore::FloatParameter(
+		"compressionThreshold",
+		"Weights below that threshold are being ignored.",
+		0.0
```

Comment:
is 0.0 a good default? Or do we want 1e-6 ? not sure...

Reply:
I think it should be pretty safe to assume that values below 1e-06 can be cut off as a default. If we ever run into visible issues in rigging, we can still set the threshold to 0 in the call.